### PR TITLE
Bugfix for IE when blur occurs on <body>

### PR DESCRIPTION
**Pull request by zmitroc**
See [description](https://www.tjvantoll.com/2013/08/30/bugs-with-document-activeelement-in-internet-explorer/)  problem

### CODE REVIEW

**Comment by anthony-redFox** [comment]
IE9 and IE10 have a bug.
If you call document.body.blur() in IE9 or IE10, you will switch application windows.

**Comment by zmitroc** [comment]
I signed it

File: dialog-polyfill.js [hunk 1]
```diff
@@ -198,7 +198,9 @@
         }).join(', ');
         target = this.dialog_.querySelector(query);
       }
-      document.activeElement && document.activeElement.blur && document.activeElement.blur();
+      if (document.activeElement && document.activeElement.blur && document.activeElement.nodeName.toLowerCase() !== "body") {
```
Comment (samthor) [inline dialog-polyfill.js]: I'm confused, we're not explicitly calling `blur` here or below. So the 2nd bug in the related document shouldn't be triggered. What am I missing?

**Comment by zmitroc** [comment]
1) Open "Explorer" or any other program.
2) Open IE9 or 10.
3) Open test.html
<strong>ACTUAL</strong>:  another program is displayed on top of IE (bug in first part (~200 line number))
<strong>EXPECTED</strong>: IE displayed on top 
4) click on the overlay
<strong>ACTUAL</strong>:  another program is displayed on top of IE (bug in second part (~384 line number))
<strong>EXPECTED</strong>: IE displayed on top 

test.html:
<pre>
&lt;!DOCTYPE html&gt;
&lt;html lang=&quot;en&quot;&gt;
&lt;head&gt;
    &lt;meta charset=&quot;UTF-8&quot;&gt;
    &lt;link rel=&quot;stylesheet&quot; href=&quot;dialog-polyfill.css&quot;&gt;
    &lt;script src=&quot;dialog-polyfill.js&quot;&gt;&lt;/script&gt;
    &lt;title&gt;&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;dialog&gt;
    I&#39;m a dialog!
    &lt;form method=&quot;dialog&quot;&gt;
        &lt;input type=&quot;submit&quot; value=&quot;Close&quot; /&gt;
    &lt;/form&gt;
&lt;/dialog&gt;
&lt;script&gt;
    var dialog = document.querySelector(&#39;dialog&#39;);
    dialogPolyfill.registerDialog(dialog);
    // Now dialog acts like a native &lt;dialog&gt;.
    setTimeout(function() {dialog.showModal()}, 1000);
&lt;/script&gt;
&lt;/body&gt;
&lt;/html&gt;
</pre>

**Comment by anthony-redFox** [comment]
Hello @samthor, any update?

**Comment by samthor** [comment]
LGTM, verified on IE9.

**Pull request merged**